app/src/main/java/com/example/chessclockk/vm/IMainActivityVM.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.chessclockk.vm
+
+import androidx.lifecycle.LiveData
+
+//TODO dodac LiveDate'y i stan
+interface IMainActivityVM {
+
+    val clockBlackLiveData: LiveData<String>
+    val clockWhiteLiveData: LiveData<String>
+
+    val stateLiveData: LiveData<MainScreenState>
+
+    fun onClockBlackPressed()
+    fun onClockWhitePressed()
+    fun onRestartClicked()
+    fun onRestartConfirmedClicked()
+    fun onPlayPauseBtnClicked()
+    fun onCustomTimeSet(customTime: String, bonus: String)
+    fun onCustomTimeSetClick()
```

Comment:
jak byłem na rozmowach dwa lata temu i teraz to widzę już w xtb, to wszyscy są zafascynowani teraz MVI zamiast MVVM. Sprowadza się to do tego, że zamiast metod masz jedną metodę do wysyłania eventu. A event to sealed klasa albo interfejs, i ma większości obiekty, a czasami data klasy